tests/abortStatus/host/host.cpp

```diff
@@ -126,7 +126,7 @@ static bool TestBasicAbort(const char* enclave_name)
             TestAbortStatus(enclave, functions[i].first, functions[i].second);
         }
 
-#ifdef OE_USE_DEBUG_MALLOC
+#if defined(OE_USE_DEBUG_MALLOC) || defined(_WIN32)
```

Comment:
Why is _WIN32 treated the same way as OE_USE_DEBUG_MALLOC?

Reply:
Latest code expects memory leaks or OK status for all scenarios after memory leaks. We are just lucky that we are not seeing memory leaks all the time with optimized builds.
So, there are no checks for platform or build type.

Reply:
My revised code allows OK or OE_MEMORY_LEAK status with terminate_enclave.